index.html

```diff
@@ -8,11 +8,11 @@
 </head>
 <body>
     <h1>todoTXT.js</h1>
-    <div id="center">
-        <button class="button" id="login" onclick="appearBtn()">Log In</button>
+    <div class="center" id="login">
+        <button class="button" id="login" onclick="appearById()">Log In</button>
```

Comment:
considering the comments in the other files, this should be called like

```suggestion
        <button class="button" id="login" onclick="appearById('modalContainer')">Log In</button>
```